paris-material/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="Paris_MaterialButton">
```

Comment:
This should only include the XML attributes that are specifically owned by the `MaterialButton` class (see https://developer.android.com/reference/com/google/android/material/button/MaterialButton), not its parents.